webpack.config.js

```diff
@@ -36,16 +36,7 @@ module.exports = {
       }, {
         test: /\.jsx?$/,
         exclude: [/node_modules/, /.*\.min\.js/],
-        use: {
-          loader: 'babel-loader',
-          options: {
-            presets: ['es2015', 'react'],
-            plugins: [
-              'transform-runtime',
-              'transform-object-rest-spread',
-            ],
-          },
-        },
+        use: 'babel-loader',
```

Comment:
The babel config was duplicated here and in `.babelrc` and should only live in the latter.